apps/polls/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+from adhocracy4.models.base import UserGeneratedContentModel
+from adhocracy4.modules import models as module_models
+
+
+class Poll(models.Model):
+    title = models.CharField(max_length=255)
+
+    module = models.ForeignKey(
+        module_models.Module,
+        on_delete=models.CASCADE,
+    )
```

Comment:
I didn't use an item, because it would pull in the userGeneratedContentModel. 
And polls are edited by admins/moderators not regular users